app/containers/wallet/WalletReceivePage.js

```diff
@@ -127,7 +128,7 @@ export default class WalletReceivePage extends Component<Props, State> {
             isHardware={wallet.isHardwareWallet}
             verify={() => actions.ada.hwVerifyAddress.verifyAddress.trigger({ wallet })}
             cancel={() => actions.ada.hwVerifyAddress.closeAddressDetailDialog.trigger()}
-            classicTheme={theme.classic}
+            classicTheme={profile.currentTheme === THEMES.YOROI_CLASSIC}
```

Comment:
What do you think of adding a method isClassic or isModern inside profile? it works fine, but it seems to verbose

Reply:
sure why not